tests/components/wiffi/test_config_flow.py

```diff
@@ -103,7 +107,28 @@ async def test_form_start_server_failed(hass, start_server_failed):
     )
 
     result2 = await hass.config_entries.flow.async_configure(
-        result["flow_id"], user_input={CONF_PORT: 8765},
+        result["flow_id"], user_input=MOCK_CONFIG,
     )
     assert result2["type"] == RESULT_TYPE_ABORT
     assert result2["reason"] == "start_server_failed"
+
+
+async def test_option_flow(hass):
+    """Test option flow."""
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_CONFIG)
+    entry.add_to_hass(hass)
+
+    assert not entry.options
+
+    result = await hass.config_entries.options.async_init(entry.entry_id, data=None)
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result["step_id"] == "init"
+
+    result = await hass.config_entries.options.async_configure(
+        result["flow_id"], user_input={CONF_TIMEOUT: 9}
+    )
+
+    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+    assert result["title"] == ""
+    assert result["data"][CONF_TIMEOUT] == 9
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5535891)